webpack.config.cjs

```diff
@@ -270,9 +270,7 @@ const webStoriesScripts = {
   },
   plugins: [
     ...sharedConfig.plugins,
-    new DependencyExtractionWebpackPlugin({
-      injectPolyfill: true,
```

Comment:
There was never really a good reason to load old polyfills mostly needed for IE11, which we don't support anyway.